### PR TITLE
Initial template

### PR DESCRIPTION
**Pull request by rizwansaeed**
First draft template containing 
* default MIT license
* issue + bug templates
* code of conduct
* contributing guide
* template README

### CODE REVIEW

**Comment by rizwansaeed** [comment]
Should we add a default set of folders for organising the repo e.g. `/src`? Having looked at a few popular repos in multiple languages there doesn't seem to be any "standard" but maybe we should do one for FINBOURNE?

e.g.

/docs
/build
/src

Thoughts?

**Comment by pms1969** [comment]
> Should we add a default set of folders for organising the repo e.g. `/src`? Having looked at a few popular repos in multiple languages there doesn't seem to be any "standard" but maybe we should do one for FINBOURNE?
> 
> e.g.
> 
> /docs
> /build
> /src
> 
> Thoughts?

I think different languages have different setups.  For instance,Rust and Golang use the src/ directory structure, while C# tends to be solution/project structured.  Not sure how python goes about it's business, and I java does that whole directory subdivision based on module names.

While I think we probably should standardise on the structure you've just proposed, we should keep some flexibility around it for different kinds of repos.

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+1. Push you changes to your fork
+1. Submit a [pull request](https://help.github.com/en/articles/about-pull-requests) to the main repository
+
+## Code review process
+
+The Core Team reviews pull requests on a regular basis and will give feedback on the corresponding issue in the repo. After feedback has been given it is expected that you respond within two weeks, the pull request may be closed if it does not show any signs of activity. An automated build will run as part of your pull request, at a minimum this needs to be passing before your contribution is accepted. 
+
+# Getting in touch
+
+Please don't use the issue tracker for support questions, for help on using LUSID please contact support@lusid.com
```
Comment (pms1969) [inline docs/CONTRIBUTING.md]: do we really want to add this to every repository as a default.  Appreciate it's completely needed for the SDK's, but other OSS projects may not even be related.  either way it has to be added individually, or removed individually, and I'm not wedded to either.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior.
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Additional context**
+Add any other context about the problem here.
```
Comment (pms1969) [inline .github/ISSUE_TEMPLATE/bug_report.md:32]: might be good to gather platform as well.  windows/linux/mac, + version of such things.

File: README.md [hunk 3]
```diff
@@ -1 +1,48 @@
...
+## Reporting Issues
+If you encounter any issues please report these the Github [issues page](https://github.com/finbourne/${REPO_NAME}/issues).
+
+---
+
+# TODOs
+
+- [ ] update repo name in template
+- [ ] set up automated build
+- [ ] set up Sonar build
```
Comment (pms1969) [inline README.md]: SonarCloud? + link to our org in said cloud service?

File: README.md [hunk 4]
```diff
@@ -1 +1,48 @@
...
+
+## Reporting Issues
+If you encounter any issues please report these the Github [issues page](https://github.com/finbourne/${REPO_NAME}/issues).
+
+---
+
+# TODOs
+
+- [ ] update repo name in template
+- [ ] set up automated build
```
Comment (pms1969) [inline README.md]: as Github Actions?  with link to GA documentation?

**Review by pms1969 — COMMENTED** [review]
Mostly just questions.  feel free to ignore.

File: docs/CONTRIBUTING.md (hunk 1 above)
Comment (rizwansaeed) [reply docs/CONTRIBUTING.md]: I had the same though and was waiting to see if anyone had a view, I've removed

File: README.md (hunk 4 above)
Comment (rizwansaeed) [reply README.md]: done

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (rizwansaeed) [reply .github/ISSUE_TEMPLATE/bug_report.md:32]: done

**Pull request merged**